test/blocks/merch/mocks/embed-utils.js

```diff
@@ -23,6 +23,9 @@ export function createTag(tag, attributes, html) {
   return el;
 }
 
+const PAGE_URL = new URL(window.location.href);
+export const SLD = PAGE_URL.hostname.includes('.aem.') ? 'aem' : 'hlx';
```

Comment:
Nit: does this actually need to be exported?

Reply:
If the tests import a module that imports SLD, it is needed yes. Since this is a replication of the utils, it should be exported similar to the utils